### PR TITLE
Add ct events

### PR DESCRIPTION
**Pull request by askpatrickw**
On hold: Looking for absentee.application deadlines...

### CODE REVIEW

**Comment by askpatrickw** [comment]
I thought fresh eyes would help... nope can't find it. I emailed the CT Elections Commission.

**Comment by askpatrickw** [comment]
Wow... Multiple email addresses listed on the CT site are bouncing...

**Comment by tannewt** [comment]
> On hold: Looking for absentee.application deadlines...

Maybe go by `Absentee ballots are to be provided by the municipal clerk beginning 31 days before an election, 21 days before a primary, or 19 days before a referendum, upon application properly made.`?

It's not super urgent since absentee is limited still.

**Comment by askpatrickw** [comment]
August 10 for both is the answer:

> From: SOTS LEAD <LEAD@ct.gov>
> Sent: Tuesday, July 7, 2020 10:13 AM
> To: Patrick Walters
> Subject: RE: Absentee Application Deadlines
> 
> yes
> 
> Legislation & Elections Administration Division
> Secretary of the State Office
> (860) 509-6271
> 
> -----Original Message-----
> From: Patrick Walters <patrickwalters@outlook.com>
> Sent: Tuesday, July 7, 2020 1:12 PM
> To: SOTS LEAD <LEAD@ct.gov>
> Subject: Re: Absentee Application Deadlines
> 
> Thank you,
> 
> And the date for the General Election on Nov 3rd?
> 
> 
> Thanks,
> Patrick
> 
> ________________________________________
> From: SOTS LEAD <LEAD@ct.gov>
> Sent: Tuesday, July 7, 2020 10:10 AM
> To: Patrick Walters
> Subject: RE: Absentee Application Deadlines
> 
> It is August 10.
> 
> 
> Legislation & Elections Administration Division Secretary of the State Office
> (860) 509-6271
> 
> -----Original Message-----
> From: Patrick Walters <patrickwalters@outlook.com>
> Sent: Tuesday, July 7, 2020 1:09 PM
> To: SOTS LEAD <LEAD@ct.gov>
> Subject: Fw: Absentee Application Deadlines
> 
> Hello,
> 
> I have a Connecticut election deadline question.
> 
> I know that when voting absentee, your completed ballot must be received by 8pm on the election day. But, what is the deadline to submit an absentee ballot application for both the August 11 Primary and the November 3rd General Election?

File: states/connecticut/elections.toml [hunk 1]
```diff
@@ -3,7 +3,68 @@ date = 2020-08-11
 name = "Presidential Primary"
 original_date = 2020-08-11
 
+[20200811.registration]
+postmarked_by = 2020-08-06
+received_by = 2020-08-06
+on_line_by = 2020-08-06
```
Comment (tannewt) [inline states/connecticut/elections.toml]:
```suggestion
online_by = 2020-08-06
```

Comment (tannewt) [reply states/connecticut/elections.toml]: I believe we've used the single underscore version before.

**Review by tannewt — CHANGES_REQUESTED** [review]

**Review by tannewt — APPROVED** [review]
Looks good! Thank you!

**Pull request merged**